### DISCUSSION TITLE
Program doesn't start, always did.

### DISCUSSION THREAD

**Discussion by PedroRdlPG** [Rem0o/FanControl.Releases#2145] [Q&A] [answered]
The program has been working fine since the beginning, but just suddenly today after the last update it won't start. I went to the Task Scheduler, right-clicked and chose 'run,' but nothing happened. I'm not sure which files to include here for you to help me.
Any solutions, does making a copy of configuration and reinstalling can work? Any other ideas? No idea what to do.

![fan](https://github.com/Rem0o/FanControl.Releases/assets/107901433/87810369-d3e2-473d-8200-232e67d984dd)

**Comment by ucalderau** [comment]
Same problem here, Fan Control updated yesterday and today I noticed my fans were too loud and found that Fan Control was not running and I now can't get it to start.

**Reply by ucalderau** [reply, marked as answer]
I manually downloaded the latest archive and it is working now, apparently the auto updater broke something.

**Reply by PedroRdlPG** [reply]
Settings remained, or not? I don't want to go on into settings

**Reply by ucalderau** [reply]
I made a backup of my previous install and copied the Settings folder to the new folder, that way it should keep all your settings and profiles.

**Reply by PedroRdlPG** [reply]
Thanks, I did a copy of UserCOnfig.json, and copied to FanControl, thanks

**Reply by shoegod49** [reply]
The auto-updater for Fan Control was flagged as Malware by Windows Defender last week for me. Now I've had nothing but problems. The same thing just happened to me today that happened to you. For some reason Fan Control doesn't load at startup and when i go to the actual exe file and click run as admin it doesn't do anything. I checked Task Manager and it's not even running in the background, it's just plain broke. I really hope I don't have to manually uninstall and reinstall this every time there's an update. I love this software but the last maybe 3 weeks it's been a real pain in the @$$.
